src/main/scala/com/microsoft/azure/cosmosdb/spark/streaming/DefaultCosmosDBWriteStreamRetryPolicyConfig.scala

```diff
@@ -32,7 +32,7 @@ class DefaultCosmosDBWriteStreamRetryPolicyConfig(configMap: Map[String, String]
     with CosmosDBLoggingTrait
     with Serializable
  {
-    val config = Config(configMap)
+    val config: Config = Config(configMap)
```

Comment:
any reason we are strictly defining types?

Reply:
This one is less necessary I agree

Reply:
I have added it in both cases because IntelliJ Scala style rules complained. My personal opinion is similar to var in C# - there is a gray area where you can argue either way whether the underlying type is clear or not. My preference is to stick with automatic enforcements unless there is an obvious readability problem (usually the exception would be to add a type). 
@moderakh - so unless you feel strongly about it I will leave it as is.